components/service-operator/config/rbac/role.yaml

```diff
@@ -4,12 +4,28 @@ apiVersion: rbac.authorization.k8s.io/v1
 kind: ClusterRole
 metadata:
   creationTimestamp: null
-  name: manager-role
+  name: {{ .Release.Name }}-service-operator-role
```

Comment:
The path of this file doesn't suggest it's part of a chart to me...

Reply:
It does not. The role is automatically generated and we have a thing in our make file that will copy and paste it into place.

We really don't want to be manually editing this, and should look into kustomize a little more... Not for MVP I don't think.

Reply:
_wontfix_